parachain/test-parachains/undying/src/lib.rs

```diff
@@ -158,5 +209,11 @@ pub fn execute(
 			post_state: hash_state(&new_state),
 		},
 		new_state,
+		messages,
 	))
 }
+
+// Returns coin probability based on hash (assuming that the hash value is indistinguishable from random)
```

Comment:
I am not sure about this. I would use a RNG and allow user to specify seed for reproducibility reasons.

Reply:
The documentation has been added.